config/config.go

```diff
@@ -204,7 +204,7 @@ type Config struct {
 	RuleFiles      []string        `yaml:"rule_files,omitempty"`
 	ScrapeConfigs  []*ScrapeConfig `yaml:"scrape_configs,omitempty"`
 
-	RemoteWriteConfig RemoteWriteConfig `yaml:"remote_write,omitempty"`
```

Comment:
Ok yeah. Another point: there's other things that are "remote" (well, Alertmanager), but they would not go under a `remote` section, so having a `remote` top-level section could also be confusing for that reason.